<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Expanse_dialog</class>
 <widget class="QDialog" name="Expanse_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size: 23px;</string>
  </property>
  <widget class="QSpinBox" name="vertical_crop">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>131</y>
     <width>149</width>
     <height>34</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="maximum">
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>21</y>
     <width>381</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>8</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font-size: 23px;</string>
      </property>
      <property name="text">
       <string>Expanse direction</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="crop_direction">
      <property name="styleSheet">
       <string notr="true">font-size: 23px;</string>
      </property>
      <item>
       <property name="text">
        <string>Center</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Upper left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Upper right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lower right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lower left</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="horizontal_crop">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>171</y>
     <width>149</width>
     <height>34</height>
    </rect>
   </property>
   <property name="maximum">
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>217</y>
     <width>151</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(75, 75,75);
color: white;</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>217</y>
     <width>131</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(75, 75,75);
color: white;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>55</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>131</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical expansion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizontal expansion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="color_button">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>90</y>
     <width>149</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>crop_direction</tabstop>
  <tabstop>color_button</tabstop>
  <tabstop>vertical_crop</tabstop>
  <tabstop>horizontal_crop</tabstop>
  <tabstop>ok_button</tabstop>
  <tabstop>cancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
